packages/vm/src/instance.rs

```diff
@@ -216,7 +216,7 @@ mod test {
 
         let handle_used = gas_before_handle - instance.get_gas();
         println!("handle used: {}", handle_used);
-        assert_eq!(handle_used, 91464);
+        assert_eq!(handle_used, 62144);
```

Comment:
Where does this come from?

Switching from 0.7 to 0.8 contract? To cosmwasm 0.7.3?
Nice in any case

Reply:
Or is it the change in how we allocate the buffers for read/write callbacks?
(Avoiding initializing 2000 0's for no real reason).

Reply:
See https://github.com/CosmWasm/cosmwasm/pull/187#discussion_r398115313